vllm/v1/core/kv_cache_manager.py

```diff
@@ -109,6 +132,9 @@ def get_computed_blocks(
             else:
                 break
 
+        self.prefix_caching_metrics["query_total"] += len(block_hashes)
+        self.prefix_caching_metrics["query_hit"] += len(computed_blocks)
```

Comment:
I see you decided to drop the great care you went to in v0 to avoid large integer values - https://github.com/vllm-project/vllm/pull/7606#discussion_r1720372929

Seems reasonable :+1:

Reply:
Yeah decided to keep it simple first and revisit later if needed.